trillian.proto

```diff
@@ -139,6 +139,10 @@ message Tree {
   // The public key used for verifying tree heads and entry timestamps.
   // Readonly.
   PublicKey public_key = 14;
+
+  // Interval after which a new signed root is produced even if there have been
+  // no submission.  If zero, this behavior is disabled.
+  int64 force_new_root_millis = 15;
```

Comment:
Maybe `max_root_duration`, where 0 means infinite? "force"-type options are usually non-default, to be used with care (at least in my experience), whereas this is perfectly fine to set.